### ISSUE TITLE
Train a coco model from scratch but the performance is very poor!

### ISSUE THREAD

**Issue by zengzengsun** [matterport/Mask_RCNN#98]
I use the code to train a coco model by the command: "python3 coco.py train --dataset=/path/to/my/datasets/coco --model=imagenet". Instead of using the train+val35K datasets, I use only train set of coco for training. However, the result is poor and only has an AP 0.082 after I finished the training.

**Comment by waleedka** [comment]
The training schedule depends on the number of GPUs, batch size, learning rate, ...etc. so it's hard to guess why you got bad results. Keep in mind that the training schedule in coco.py is an example. There is a comment that mentions that in the code. The new release 2 schedule is closer to what you might want to use, but it's also an example. You need to adjust the training schedule depending on your setup.

**Comment by ash1995** [comment]
@zengzengsun  Were you able to find out why your AP was low? I'm facing the same problem (`Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.075`)
